CMSgov/design-system
# [WNMGDS-1637] Implement native dialog component

## Summary

[JIRA ticket](https://jira.cms.gov/browse/WNMGDS-1637)

- [x] Create `<NativeDialog />`
- [x] Add polyfill support
- [x] Implement in `<Drawer />`
    - [x] Markup
    - [x] A11y
    - [x] Behavior (focus trapping, interaction with background content, etc.)

### Testing (building)

- Checkout the [docs site](https://design-system-demo.s3.amazonaws.com/WNMGDS-1637/implement-native-dialog/core/components/drawer/index.html) to make sure Drawer examples look/behavior like [PROD](https://design.cms.gov/components/drawer/) (Note: You'll need to add `index.html` to the end of each iframe URL example to see them 😢 )
- Check out [Storybook](https://design-system-demo.s3.amazonaws.com/WNMGDS-1637/implement-native-dialog/core/storybook/index.html?path=/story/components-drawer--drawer-toggle-with-drawer) for more look/behavior checks (optional)
